File: backend/components/metiers/administration_publique/utils/validators/samples/sample_usage.js
// sample_usage.js
// Exemple d'utilisation du module validators (JS)
const validators = require('../core/validators');
const data = require('./sample_validators_data.json');

// Validation d'email
console.log('Email valide ?', validators.validateEmail(data.email));

// Validation d'un champ obligatoire
console.log('Champ obligatoire valide ?', validators.validateRequired(data.requiredField));

File: backend/components/metiers/administration_publique/utils/validators/samples/sample_usage.py
# sample_usage.py
"""Exemple d'utilisation du module validators (Python)"""
from ..core import validators
import json
with open('sample_validators_data.json') as f:
    data = json.load(f)

print('Email valide ?', validators.validate_email(data['email']))
print('Champ obligatoire valide ?', validators.validate_required(data['requiredField']))
